tools/alive-tv.cpp

```diff
@@ -269,6 +255,8 @@ llvm::Function *findFunction(llvm::Module &M, const string &FName) {
 }
 }
 
+void optimizeModule(llvm::Module *M, llvm::StringRef optArgs);
```

Comment:
Aha I see what you mean! Thanks for the explanation!

Reply:
should I add the function into llvm_util namespace?

Reply:
yes, thanks!

You can't do it with StringRef because it is passed by value. Alternatively, you can use C++'s string_view. We don't usually use LLVM's stuff so we stay as independent as possible.

Reply:
Done!